meilisearch-release-changelog/v1.14.0.md

```diff
@@ -0,0 +1,187 @@
+* Introduce `exhaustiveFacetCount` parameter to `/facet-search` route to retrieve an exhaustive facet count by @ManyTheFish in [#5369](https://github.com/meilisearch/meilisearch/pull/5369)
+* Reduce RAM consumption of arroy by @irevoire in https://github.com/meilisearch/arroy/pull/105
+* Experimental feature: Cache embeddings during search (Read more in the [feature discussion](https://github.com/orgs/meilisearch/discussions/818)) by @dureuill in [#5418](https://github.com/meilisearch/meilisearch/pull/5418)
+* Extend batch progress view to include indexing of vectors by @irevoire in [#5420](https://github.com/meilisearch/meilisearch/pull/5420)
+
+# Fixes 🐞
+
+* Geo update bug by @ManyTheFish in https://github.com/meilisearch/meilisearch/pull/5407
+* Isolate word fst usage by @ManyTheFish in https://github.com/meilisearch/meilisearch/pull/5415
+* Fix CI to work with merge queues by @Kerollmops in https://github.com/meilisearch/meilisearch/pull/5456
+* Support fetching the pooling method from the model configuration by @dureuill in [#5355](https://github.com/meilisearch/meilisearch/pull/5355)
+* Delete unused prefixes by @Kerollmops in [#5413](https://github.com/meilisearch/meilisearch/pull/5413)
+
+# Misc
+
+* Dependencies updates
+  * Bump Ubuntu in the CI from 20.04 to 22.04 by @Kerollmops in [#5338](https://github.com/meilisearch/meilisearch/pull/5338)
+  * Bump heed to v0.22 by @irevoire and @Kerollmops in [#5406](https://github.com/meilisearch/meilisearch/pull/5406)
+  * Bump ring to v0.17.14 to compile on old aarch64 by @Kerollmops in https://github.com/meilisearch/meilisearch/pull/5423
+  * Bump zip from 2.2.2 to 2.3.0 by @dependabot in https://github.com/meilisearch/meilisearch/pull/5426
+  * Update Charabia v0.9.3 by @ManyTheFish in https://github.com/meilisearch/meilisearch/pull/5454
+  * Update mini-dashboard to v0.2.19 version by @curquiza in https://github.com/meilisearch/meilisearch/pull/5436
+* CIs and tests
+  * Improve test performance of get_index.rs  by @DerTimonius in [#5210](https://github.com/meilisearch/meilisearch/pull/5210)
+  * Ollama Integration Tests by @Kerollmops in [#5308](https://github.com/meilisearch/meilisearch/pull/5308)
+  * Ensure the settings routes are properly configured when a new field is added to the Settings struct by @MichaScant in [#5149](https://github.com/meilisearch/meilisearch/pull/5149)
+  * Skip a snapshot test on Windows by @Kerollmops in [#5383](https://github.com/meilisearch/meilisearch/pull/5383)
+* Accept total batch size in human size by @Kerollmops in https://github.com/meilisearch/meilisearch/pull/5421
+* Avoid reindexing searchable order changes by @ManyTheFish in https://github.com/meilisearch/meilisearch/pull/5402
+* Add more progress levels to measure merging by @Kerollmops in https://github.com/meilisearch/meilisearch/pull/5422
+> ⚠️ When working with it in the doc or the integrations, please consider that the content can change anytime. It must not be enforced.
```

Comment:
I'm not sure it means something for the users to say "When working with it in the doc or the integrations". I'm not even sure to understand what we mean here